queries.sql

```diff
@@ -9,3 +9,79 @@ SELECT DISTINCT * FROM animals WHERE neutered = true;
 SELECT DISTINCT * FROM animals WHERE name <> 'Gabumon';
 SELECT DISTINCT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
```

Comment:
- [ ] Kindly add the screenshot of this transaction to your pull request summary. Remember that you should add screenshots of the transactions to verify that your code works according to the requirement. 👍

Reply:
![Screenshot (180)](https://github.com/AleWaweru/vet-clinic-database/assets/102799430/ac02fbfe-1cc1-4782-8079-6594d10de0da)